src/client/game.cpp

```diff
@@ -2620,8 +2620,8 @@ void Game::updateCameraDirection(CameraOrientation *cam, float dtime)
 	Since Minetest has its own code to synthesize mouse events from touch events,
 	this results in duplicated input. To avoid that, we don't enable relative
 	mouse mode if we're in touchscreen mode. */
-	if (cur_control)
-		cur_control->setRelativeMode(!g_touchscreengui && !isMenuActive());
+	if (!g_touchscreengui && cur_control)
+		cur_control->setRelativeMode(!isMenuActive());
```

Comment:
This change is wrong, it will prevent disabling relative mouse mode if we implement a way to toggle touchscreen mode in-game. Why did you do it?

Reply:
sorry about that, it was the debugging process of non-working digging on Android and I kept the change. Removed.